e2e-tests/docs/CI.md

```diff
@@ -0,0 +1,135 @@
+# CI Steps and Processes for Testing
+
+This document provides a comprehensive overview of our Continuous Integration (CI) testing processes. It explains when and how different test jobs are triggered, where to monitor them, and details about the environments hosting these tests, along with guidance on how to access them.
+
+## GitHub Pull Requests
+
+When a new Pull Request (PR) is opened at [backstage-showcase](https://github.com/janus-idp/backstage-showcase), tests can be triggered in two ways:
+
+1. **Commenting `/ok-to-test`**: Only members of the janus-idp GitHub organization can mark the PR with this comment to validate it for testing.
+
+2. **Triggering Tests Post-Validation**: After a janus-idp member has validated the PR, anyone can trigger tests using the following commands:
+   - `/test` or `/test all`
+   - `/retest`
+
+These interactions are picked up by the OpenShift-CI service, which sets up a test environment on the **IBM Cloud**, specifically on the `rhdh-pr-os` OpenShift Container Platform (OCP) cluster. The configurations and steps for setting up this environment are defined in the `openshift-ci-tests.sh` script.
```

Comment:
Would be worth to consider adding a 2-3 high level sentences explaining how the script works - something like high level flow.

Reply:
Do we need to mention the detailed info and names of clusters that we use? This information would be useful for internal users but I think it would be a little too much detail. Also, when we need to use new clusters in the future the cluster name would be changed as well.

Reply:
done! @psrna  openshift-ci-tests.sh overview added at the bottom of the file. Also references its path there and its overview on its previous mentions.
@josephca  The clusters names have been removed, considering that they will change and that is not relevant on a public-facing document indeed.

Thank you both for your review!